accepted/0032-scene-wizard.md

```diff
@@ -0,0 +1,36 @@
+- Start Date: 2020-09-10
+- RFC PR: #32
+
+# Summary
+
+This proposal is for adding a scene wizard for new users when first loading OBS.
+
+# Motivation
+
+Users often find it difficult to set up OBS.
+
+# Detailed Design
+
+The user will also be able to open the wizard in the scene collection menu.
+
+### Wizard page 1
+- Buttons that allow the user to select how they want to use OBS
```

Comment:
I think stylised buttons would be good - big icon, short text. Make it SUPER user-friendly - this'll be their first interaction with the program.